src/sass/components/_header.scss

```diff
@@ -178,6 +178,13 @@ header {
         display: none;
       }
 
+      %category-hover {
```

Comment:
Okej, ale nie widzę wykorzystania tego. Gdzie tego używasz?

Reply:
Używam w tym samym pliku w linijkach 210 i 267. W obu chciałem ten sam efekt i wykorzystałem % aby nie powtarzać kodu. Przy review innego taska również zwrócono mi na to uwagę. Czy to znaczy, że powinienem zastosować inne rozwiązanie?

Reply:
moja wina - nie widziałem tego